Symfony Insights tests failing

:clock11: **When did the problem start?**

Not 100% sure but they've been failing for at least a week

:camera: **Live badge**

<!-- Provide a link to the live badge in plain text and markdown.  -->

All our test and example badges seem to be failing e.g:
https://img.shields.io/symfony/i/grade/45afb680-d4e6-4e66-93ea-bcfa79eb8a87

:wrench: **Is the live badge working?**

I have managed to find an example of a working badge "in the wild" e.g:
https://img.shields.io/symfony/i/grade/15c5c748-f8d8-4b56-b536-a29a151aac6c?color=success

:link: **CircleCI link**

https://app.circleci.com/pipelines/github/badges/daily-tests/773/workflows/b4870372-91f1-45a7-ad33-76d927034d16/jobs/1671/tests#failed-test-8

:beetle: **Stack trace**

```
AssertionError: label mismatch: expected 'symfony insight' to equal 'grade'
    at Function._expectField (core/service-test-runner/icedfrisby-shields.js:69:51)
    at IcedFrisbyNock.<anonymous> (core/service-test-runner/icedfrisby-shields.js:53:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:954:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1003:24)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:1017:7
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:976:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1276:20)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**

Suspect we need new example projects. I think the code/service is ok.

--- a/services/symfony/sensiolabs-redirect.tester.js
+++ b/services/symfony/sensiolabs-redirect.tester.js
@@ -8,5 +8,5 @@ const t = (module.exports = new ServiceTester({
 }))
 
 t.create('sensiolabs insight')
-  .get('/i/45afb680-d4e6-4e66-93ea-bcfa79eb8a87.svg')
-  .expectRedirect('/symfony/i/grade/45afb680-d4e6-4e66-93ea-bcfa79eb8a87.svg')
+  .get('/i/15c5c748-f8d8-4b56-b536-a29a151aac6c.svg')
+  .expectRedirect('/symfony/i/grade/15c5c748-f8d8-4b56-b536-a29a151aac6c.svg')

--- a/services/symfony/symfony-insight-grade.service.js
+++ b/services/symfony/symfony-insight-grade.service.js
@@ -16,7 +16,7 @@ module.exports = class SymfonyInsightGrade extends SymfonyInsightBase {
     {
       title: 'SymfonyInsight Grade',
       namedParams: {
-        projectUuid: '45afb680-d4e6-4e66-93ea-bcfa79eb8a87',
+        projectUuid: '15c5c748-f8d8-4b56-b536-a29a151aac6c',
       },
       staticPreview: this.render({
         grade: 'bronze',

--- a/services/symfony/symfony-insight-stars.service.js
+++ b/services/symfony/symfony-insight-stars.service.js
@@ -25,7 +25,7 @@ module.exports = class SymfonyInsightStars extends SymfonyInsightBase {
     {
       title: 'SymfonyInsight Stars',
       namedParams: {
-        projectUuid: '45afb680-d4e6-4e66-93ea-bcfa79eb8a87',
+        projectUuid: '15c5c748-f8d8-4b56-b536-a29a151aac6c',
       },
       staticPreview: this.render({
         grade: 'silver',

--- a/services/symfony/symfony-insight-violations.service.js
+++ b/services/symfony/symfony-insight-violations.service.js
@@ -12,7 +12,7 @@ module.exports = class SymfonyInsightViolations extends SymfonyInsightBase {
     {
       title: 'SymfonyInsight Violations',
       namedParams: {
-        projectUuid: '45afb680-d4e6-4e66-93ea-bcfa79eb8a87',
+        projectUuid: '15c5c748-f8d8-4b56-b536-a29a151aac6c',
       },
       staticPreview: this.render({
         numViolations: 0,

--- a/services/symfony/symfony-insight-violations.tester.js
+++ b/services/symfony/symfony-insight-violations.tester.js
@@ -11,6 +11,6 @@ t.create('valid project violations')
   .expectBadge({
     label: 'violations',
     message: withRegex(
-      /\d* critical|\d* critical, \d* major|\d* critical, \d* major, \d* minor|\d* critical, \d* major, \d* minor, \d* info|\d* critical, \d* minor|\d* critical, \d* info|\d* major|\d* major, \d* minor|\d* major, \d* minor, \d* info|\d* major, \d* info|\d* minor|\d* minor, \d* info/
+      /0|\d* critical|\d* critical, \d* major|\d* critical, \d* major, \d* minor|\d* critical, \d* major, \d* minor, \d* info|\d* critical, \d* minor|\d* critical, \d* info|\d* major|\d* major, \d* minor|\d* major, \d* minor, \d* info|\d* major, \d* info|\d* minor|\d* minor, \d* info/
     ),
   })

--- a/services/symfony/symfony-test-helpers.js
+++ b/services/symfony/symfony-test-helpers.js
@@ -1,6 +1,6 @@
 'use strict'
 
-const sampleProjectUuid = '45afb680-d4e6-4e66-93ea-bcfa79eb8a87'
+const sampleProjectUuid = '15c5c748-f8d8-4b56-b536-a29a151aac6c'
 
 function createMockResponse({ status = 'finished', grade, violations }) {
   let response = `

